### ISSUE TITLE
Cannot find @microsoft/generator-sharepoint

### ISSUE THREAD

**Issue by brooksca** [pnp/generator-spfx#95]
#### General Information
* [x] Usage
* [ ] Development
* [ ] Documentation
* [ ] Feature Request

#### Specific generator
* [ ] Addons
* [ ] HandlebarsJS

#### What Version you are running?
* NodeJS: 8.12.0
* NPM: 6.4.1
* @pnp/spfx: 1.2.0

#### Describe your problem
When I run `yo @pnp/spfx`, I get the following error:

```
module.js:550
    throw err;
    ^

Error: Cannot find module '../node_modules/@microsoft/generator-sharepoint/lib/generators/app/index.js'
    at Function.Module._resolveFilename (module.js:548:15)
    at Function.resolve (internal/module.js:18:19)
    at Object.<anonymous> (C:\Users\[username]\AppData\Local\Yarn\Data\global\node_modules\@pnp\generator-spfx\app\subGenerators.js:9:21)
    at Module._compile (module.js:653:30)
    at Object.Module._extensions..js (module.js:664:10)
    at Module.load (module.js:566:32)
    at tryModuleLoad (module.js:506:12)
    at Function.Module._load (module.js:498:3)
    at Module.require (module.js:597:17)
    at require (internal/module.js:11:18)
```
Windows 10 Pro (Build 1803)

I've tried downgrading `@pnp/spfx` to 1.0.0, but no joy. 

Was able to get a local version up and running by amending subGenerators.js to:
```
module.exports = {
    // Add additional addons
    'addons': require.resolve('../generators/addons'),
    // Handlebars
    'handlebars': require.resolve('../generators/handlebars'),
    //VueJs
    'vuejs': require.resolve('../generators/vuejs'),
    // SPFx
    // Previous: 'spfx': require.resolve('../node_modules/@microsoft/generator-sharepoint/lib/generators/app')
    'spfx': require.resolve('@microsoft/generator-sharepoint/lib/generators/app')
}
```

**Comment by phawrylak** [comment]
I confirm that this error occurs when you install package using yarn. If you install package using npm everything is ok. You can probably create PR with your fix. 👍

**Comment by StfBauer** [comment]
@phawrylak Have you tested it with NPM 6.0? AFAIK npm 6.0 is currently not supported yet.
@brooksca Thank you for letting us know! As an intermediate solution you can for the usage of NPM by using the command line switch `--package-manager npm` or `--pm npm`

Since we bundle the default SPFx generator with this generator I guess this workaround should work need to test it.

**Comment by StfBauer** [comment]
Should be fixed in Version 1.3.0 - Can you please confirm?
